hildon/hildon-gtk.c

```diff
@@ -31,6 +31,24 @@
 #include "hildon-edit-toolbar-private.h"
 #include "hildon-private.h"
 
+// Add placeholder implementations for Maemo-specific Gtk functions
```

Comment:
Please try to keep the style of the package you're making changes to.
This package have comments of the form /* */ in most cases.

Reply:
This comment seems to be a good commit message.
I don't think it's really needed here.

Reply:
It's useful in the code - you don't usually look at the commit history while analyzing the code.

Reply:
And I changed the comment style